### PR TITLE
DOC: installation instructions

### PR DESCRIPTION
**Pull request by dmgav**
Updated installation instructions: options to install from `conda-forge` or `nsls2forge`.

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -31,25 +31,39 @@ identical for all supported OS.
...
+       conda install pyxrf -c conda-forge
+
+   PyXRF package is also maintained at :code:`nsls2forge` Anaconda channel.
+   Installation from :code:`nsls2forge` involves similar steps:
+
+   .. code:: python
+ 
+       conda create -n pyxrf-env python=3.7
+       conda activate pyxrf-env
        conda install pyxrf atom=0.4.3 -c nsls2forge
```
Comment (mrakitin) [inline docs/installation.rst:63]:
```suggestion
       conda install pyxrf -c nsls2forge
```

File: docs/installation.rst [hunk 2]
```diff
@@ -31,25 +31,39 @@ identical for all supported OS.
...
        conda activate pyxrf-env
 
-5. Install PyXRF in the active Conda environment (in our example :code:`pyxrf-env`):
+   Install PyXRF:
 
    .. code:: python
 
+       conda install pyxrf -c conda-forge
+
+   PyXRF package is also maintained at :code:`nsls2forge` Anaconda channel.
```
Comment (mrakitin) [inline docs/installation.rst:56]:
```suggestion
   PyXRF package is also maintained at the :code:`nsls2forge` Anaconda channel.
```

File: docs/installation.rst [hunk 3]
```diff
@@ -31,25 +31,39 @@ identical for all supported OS.
       If PyXRF is installed in an older Conda environment then different command syntax 
       must be used on some platforms (particularly Windows). Refer to the Conda documentation
       for more details.
-   
-3. Create a new Conda environment (e.g. :code:`pyxrf-env`) with Python 3.6 or 3.7:
+
+3. Install PyXRF from :code:`conda-forge` or :code:`nsls2forge` Anaconda channel.
+
+   PyXRF is currently available from :code:`conda-forge` Anaconda channel. Create new
```
Comment (mrakitin) [inline docs/installation.rst:37]:
```suggestion
   PyXRF is currently available from the :code:`conda-forge` Anaconda channel. Create new
```

File: docs/installation.rst [hunk 4]
```diff
@@ -31,25 +31,39 @@ identical for all supported OS.
       If PyXRF is installed in an older Conda environment then different command syntax 
       must be used on some platforms (particularly Windows). Refer to the Conda documentation
       for more details.
-   
-3. Create a new Conda environment (e.g. :code:`pyxrf-env`) with Python 3.6 or 3.7:
+
+3. Install PyXRF from :code:`conda-forge` or :code:`nsls2forge` Anaconda channel.
```
Comment (mrakitin) [inline docs/installation.rst:35]:
```suggestion
3. Install PyXRF from :code:`conda-forge` or :code:`nsls2forge` Anaconda channels.
```

File: docs/installation.rst [hunk 5]
```diff
@@ -31,25 +31,39 @@ identical for all supported OS.
...
+   PyXRF package is also maintained at :code:`nsls2forge` Anaconda channel.
+   Installation from :code:`nsls2forge` involves similar steps:
+
+   .. code:: python
+ 
+       conda create -n pyxrf-env python=3.7
+       conda activate pyxrf-env
        conda install pyxrf atom=0.4.3 -c nsls2forge
 
+   It is not recommended to mix packages from :code:`conda-forge` channel and :code:`nsls2forge`/defaults
```
Comment (mrakitin) [inline docs/installation.rst:65]:
```suggestion
   It is not recommended to mix packages from :code:`conda-forge` channel and :code:`nsls2forge`/:code:`defaults`
```

File: docs/installation.rst [hunk 6]
```diff
@@ -84,7 +98,14 @@ Updating PyXRF
        conda activate pyxrf-env
 
 
-3. Update PyXRF by typing:
+3. Update PyXRF from the same Anaconda channel (:code:`conda-forge` or :code:`nsls2forge`) that
```
Comment (mrakitin) [inline docs/installation.rst:101]:
```suggestion
4. Update PyXRF from the same Anaconda channel (:code:`conda-forge` or :code:`nsls2forge`) that
```

**Review by mrakitin — COMMENTED** [review]
Looks good. Suggested some copy-editing.

File: docs/installation.rst (hunk 1 above)
Comment (dmgav) [reply docs/installation.rst:63]: I remember sometimes it didn't install the right version of atom. This is why the recommendation to specify the version is there.

Comment (mrakitin) [reply docs/installation.rst:63]: Should it be for the `conda-forge` case too?

Comment (dmgav) [reply docs/installation.rst:63]: I didn't see the problem. Let me check. May be it can be removed from `nsls2forge` instructions.

Comment (dmgav) [reply docs/installation.rst:63]: Ok, let's remove `atom=0.4.3` from instructions and see if anyone complains.

**Review by mrakitin — APPROVED** [review]
Looks good to me now.

**Pull request merged**